utils/pdf/BillingTemplate.js

```diff
@@ -0,0 +1,383 @@
+import React from 'react';
+import {
+  Page,
+  Text,
+  View,
+  StyleSheet,
+  Document,
+  Image,
+  Font
+} from '@react-pdf/renderer';
+
+Font.register({
+  family: 'Open Sans',
+  src: './assets/fonts/OpenSans-Regular.ttf'
+});
+Font.register({
+  family: 'Open Sans Bold',
+  src: './assets/fonts/OpenSans-Bold.ttf'
+});
+
+const styles = StyleSheet.create({
+  backgroundWhite: {
+    backgroundColor: 'white'
+  },
+  gray10: {
+    color: '#747474'
```

Comment:
i think it's a good idea to start a `main.css` or `colors.css` file now to avoid having to deal with consistency issues down the road. Ditto for all hex codes declared below

Reply:
i guesssssss so :P 

Will have to ask Ashley why all these styles aren't actually in a css file speaking of which

Reply:
ahh frk if it's in a css file i'd rather it be in kebab case. but it's fine, we can address that in a future PR